js/views/PageNav.js

```diff
@@ -20,6 +20,7 @@ export default class extends View {
         'click .js-navMin': 'navMinClick',
         'click .js-navMax': 'navMaxClick',
         'keyup .js-addressBar': 'onKeyupAddressBar',
+        'mouseup .js-addressBar': 'onMouseUpAddressBar',
```

Comment:
I think it's much simpler to just use the `focusin` event instead of the mouseup, then you don't have to do all this checking to see if it's selected.